explainer.py

```diff
@@ -0,0 +1,164 @@
+import os
+import time
+import json
+import asyncio
+from datetime import datetime
+from presentation_parser import PresentationParser
+from gpt_slide_explainer import GPTSlideExpander
+from db.db_engine import Upload, UploadStatus, get_engine
+from sqlalchemy.orm import Session
+
+UPLOADS_FOLDER = 'uploads'
+OUTPUTS_FOLDER = 'outputs'
+STATUS_FILE = 'status.json'
+SLEEP_TIME = 10  # seconds
+
+
+def process_files():
```

Comment:
Separate code to small functions.  Read more: https://softwareengineering.stackexchange.com/questions/308108/when-is-it-appropriate-to-make-a-separate-function-when-there-will-only-ever-be